addons/mail/static/src/new/discuss/discuss.xml

```diff
@@ -3,7 +3,7 @@
 
 <t t-name="mail.discuss" owl="1">
     <div class="o-mail-discuss d-flex h-100 flex-grow-1" t-att-class="{ 'o-mail-device-is-small flex-column align-items-center bg-view': store.isSmall }">
-        <Sidebar t-if="!props.public and !store.isSmall"/>
+        <t t-name="sidebar"/>
```

Comment:
This doesn't look great, I would prefer to target something existing if possible (eg. "before topbar" or "inside root (at the start)").